cmake/kokkos_arch.cmake

```diff
@@ -91,10 +91,11 @@ IF(Kokkos_ENABLE_HIP OR Kokkos_ENABLE_OPENMPTARGET OR Kokkos_ENABLE_UNSUPPORTED_
   SET(KOKKOS_SHOW_HIP_ARCHS ON)
 ENDIF()
 
-KOKKOS_ARCH_OPTION(VEGA900         GPU  "AMD GPU MI25 GFX900"      "KOKKOS_SHOW_HIP_ARCHS")
-KOKKOS_ARCH_OPTION(VEGA906         GPU  "AMD GPU MI50/MI60 GFX906" "KOKKOS_SHOW_HIP_ARCHS")
-KOKKOS_ARCH_OPTION(VEGA908         GPU  "AMD GPU MI100 GFX908"     "KOKKOS_SHOW_HIP_ARCHS")
-KOKKOS_ARCH_OPTION(VEGA90A         GPU  "AMD GPU MI200 GFX90A"     "KOKKOS_SHOW_HIP_ARCHS")
+KOKKOS_ARCH_OPTION(VEGA900         GPU  "AMD GPU MI25 GFX900"           "KOKKOS_SHOW_HIP_ARCHS")
+KOKKOS_ARCH_OPTION(VEGA906         GPU  "AMD GPU MI50/MI60 GFX906"      "KOKKOS_SHOW_HIP_ARCHS")
+KOKKOS_ARCH_OPTION(VEGA908         GPU  "AMD GPU MI100 GFX908"          "KOKKOS_SHOW_HIP_ARCHS")
+KOKKOS_ARCH_OPTION(VEGA90A         GPU  "AMD GPU MI200 GFX90A"          "KOKKOS_SHOW_HIP_ARCHS")
+KOKKOS_ARCH_OPTION(VEGA1100        GPU  "AMD GPU RX 7900 XT(X) GFX1100" "KOKKOS_SHOW_HIP_ARCHS")
```

Comment:
Isn't gfx1100 "Navi" instead of "Vega"?

Reply:
You're absolutely right, very blatant oversight TBH.